src/components/SearchResults/SearchResults.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import { users as fakeUsers} from '../../assets/fake-users'
+import { UserResults } from '../../assets/definitions'
```

Comment:
I think the addition of a definitions file will keep our code clean and organized, and allow us to reuse definitions! Great job.

Reply:
Me too! Also made #29 for more of this!